example/js/example.js

```diff
@@ -123,6 +125,7 @@ var ContextMenu = function(blueprint3d) {
     $("#item-width").val(cmToIn(selectedItem.getWidth()).toFixed(0));
     $("#item-height").val(cmToIn(selectedItem.getHeight()).toFixed(0));
     $("#item-depth").val(cmToIn(selectedItem.getDepth()).toFixed(0));
+    $("#item-name").val(selectedItem.name);
```

Comment:
What is the intent of this name field? How is it different than item.metadata.itemName?